scss/minibar.scss

```diff
@@ -11,7 +11,7 @@ $element-hover-bg: lighten($color-a-dark, 5%);
      @extend .rex-minibar-icon--fa-window-minimize;
  }
 
-.rex-minibar {
+:host {
```

Comment:
da das css jetzt direkt in der komponente gerendert wird, muss man "von innen nach außen" stylen.

dies scheint hier aber so nicht richtig zu sein.. ich hab das hier so gemacht, inspiriert von https://developers.google.com/web/fundamentals/web-components/shadowdom

@schuer @tbaddade @bloep hast jmd vllt eine idee wie es sein müsste?

Reply:
Ich würde hier einfacheres CSS schreiben und kein SCSS, da es mit den `&` nicht richtig kompiliert wird und so falsches CSS entsteht.

**Alt**

```scss
:host {
    position: fixed;
    
    &[data-minibar="false"] {
        height: $minibar-height;
    }
}
```

**Neu**

```css
--minibar-height: 36px;
:host {
    position: fixed;
}
:host([data-minibar="false"]) {
    height: var(--minibar-height);
}
```